src/test/index.ts

```diff
@@ -16,12 +16,16 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as os from 'os';
 import type { LaunchOptions, BrowserContextOptions, Page } from '../../types/types';
 import type { TestType, PlaywrightTestArgs, PlaywrightTestOptions, PlaywrightWorkerArgs, PlaywrightWorkerOptions } from '../../types/test';
 import { rootTestType } from './testType';
-
+import { removeFolders } from '../utils/utils';
 export { expect } from './expect';
 export const _baseTest: TestType<{}, {}> = rootTestType.test;
+
+const artifactsFolder = path.join(os.tmpdir(), 'playwright-test-artifacts-');
```

Comment:
this looks like it should use mkdrtmp to generate some random characters after 'playwright-test-artifacts-'. Right now it will race if two tests run in parallel.